testdata/sqllogictests/delete.slt

```diff
@@ -0,0 +1,22 @@
+# Tests for deleting rows from a table
+
+statement ok
+create table t1 as select * from generate_series(1, 15, 2);
+
+statement ok
+delete from t1 where generate_series > 7;
+
+query I
+select * from t1;
+----
+1
+3
+5
+7
+
+statement ok
+delete from t1;
+
+query I
+select * from t1
+----
```

Comment:
Can we add more tests here?

* Error when no table given/many tables given
* Some more complex where clauses

Reply:
Added the tests